tcpdf.php

```diff
@@ -16389,6 +16389,8 @@ public function getHTMLFontUnits($val, $refsize=12, $parent_size=12, $defaultuni
 	 * @since 3.2.000 (2008-06-20)
 	 */
 	protected function getHtmlDomArray($html) {
+		if(empty($html))
+			return [];
```

Comment:
```suggestion
			return array(); 
```
IIRC we still have to support old php versions